prisma/schema.prisma

```diff
@@ -16,7 +16,7 @@ datasource db {
 
 model Student {
     studentId     String @id @default(cuid())
-    studentCardId String
+    studentCardId String @unique
```

Comment:
We'll need create a DB migration for this later. No issues for now!

Reply:
Done